content/en/docs/cli-reference/notation.md

```diff
@@ -0,0 +1,10 @@
+---
+title: notation
+---
+Notation - a tool to sign and verify artifacts
+
```

Comment:
Should we keep this file? It doesn't have substantial content.

Reply:
The output of `notation` and `notation -h` is the same.